tower-rate-limit/tests/rate_limit.rs

```diff
@@ -3,16 +3,18 @@ extern crate tower_mock;
 extern crate tower_rate_limit;
 extern crate tower_service;
 extern crate tokio_timer;
+extern crate tokio;
```

Comment:
It was necessary to update these tests to use `tokio`, as the new `tokio_timer` wouldn't work otherwise. This is an unrelated change, but was necessary for the `rate-limit` crate to compile.